test-app-reactive/src/test/kotlin/uk/gov/justice/digital/hmpps/testappreactive/integration/auth/JwtAuthorisationHelperBeanTest.kt

```diff
@@ -21,6 +22,7 @@ import java.security.interfaces.RSAPublicKey
   properties = ["hmpps.test.jwt-helper-enabled=false"],
 )
 @ActiveProfiles("test")
+@AutoConfigureWebTestClient
```

Comment:
`@SpringBootTest` doesn't automatically now inject a web client, so need to add one explicitly